### ISSUE TITLE
another version hessian2 support

### ISSUE THREAD

**Issue by wbpcode** [alibaba/hessian2-codec#32]
A helpless fact is that there are two different versions of hessian2. One is the version that the dubbo used. See http://hessian.caucho.com/doc/hessian-ws.html. 

Another is the version the sofa used. See https://www.caucho.com/resin-3.1/doc/hessian-2.0-spec.xtp

They are incompatible. :(

Will the community accepts the related implementation of the another version?
cc @zyfjeff

**Comment by zyfjeff** [comment]
These two are not the same thing, the former is the definition of a Hessian Web Service, and the latter is the serialization of the base type of Hessian

**Comment by wbpcode** [comment]
Sorry, I pasted error link in the description. Updated. :)

**Comment by zyfjeff** [comment]
Obviously, sofa uses the draft version of the hessian2 spec, which is not officially supported, and we should not deviate from the official spec for a long time, sofa should be compatible with the official spec

**Comment by wbpcode** [comment]
Ideally, yeah. But, I think it's hard (and maybe impossible?) for `sofa` to do this update. Because this require all `sofa` services in the cluster to update the codec lib version at same time. These is no any compatibility between these two versions. orz.
